02-data-operations.Rmd

```diff
@@ -0,0 +1,239 @@
+
+# Data operations {#Data-operations}    
+
+The [`SummarizedExperiment`](https://bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html) (`SE`) is a widely used class for analyzing data obtained by common sequencing techniques. `SE` is common structure for several Bioconductor packages that are used for analyzing RNAseq, ChIp-Seq data. `SE` class is also used in R packages for analyzing microarrays, flow cytometry, proteomics, single-cell sequencing data and many more. The single-cell analysis is facilitated by [SingelCellExperiment class](https://bioconductor.org/packages/release/bioc/vignettes/SingleCellExperiment/inst/doc/intro.html) (`SCE`), which allows the user to store results of dimensionality reduction and alternative experiments. Alternative experiments (`altExps`) can be differently processed data within the analysis workflows. 
+
+Recently, [`TreeSummarizedExperiment`](https://www.bioconductor.org/packages/release/bioc/html/TreeSummarizedExperiment.html) (`TSE`), [`MicrobiomeExperiment`](https://github.com/FelixErnst/MicrobiomeExperiment) were developed to extend the `SE` and `SCE` class for incorporating hierarchical information (including phylogenetic tree) and reference sequences.    
+
+The `mia` package implements tools using these classes for analysis of microbiome sequencing data.       
+
+## Common terminologies    
+
+`assays`     : This slot is similar to `otu_table` in `phyloseq`. In a `SummarizedExperiment`
+               object multiple assays, raw counts, transformed counts can be stored. See also 
+               [`MultiAssayExperiment`](https://bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html) for storing data from multiple `assays` such as RNASeq, Proteomics, etc.       
```

Comment:
I think this wrong. `MultiAssayExperiments` uses the `experiments` accessor

Reply:
Maybe drop the `MAE` reference for now

Reply:
Thanks for fixing this.